packages/itwin/reports-config-widget/src/widget/components/ReportsContainer.scss

```diff
@@ -0,0 +1,19 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+@import '~@itwin/itwinui-css/scss/variables';
+
+.nz-widget-contentRenderer {
```

Comment:
where is this being used?

Reply:
Dirty fix to some scroll issue that I saw others implement for the widget's container within the viewer. I'll get rid of it and see if it comes back.